Upstream-Status: Pending

This patch fixes parallel install issue that lib libpixbufloader-png.la
depends on libgdk_pixbuf-2.0.la which will be regenerated during insta-
llation, if libgdk_pixbuf-2.0.la is regenerating and at the same time
libpixbufloader-png.la links it, the error will happen.

Error message is:
* usr/bin/ld: cannot find -lgdk_pixbuf-2.0
* collect2: ld returned 1 exit status

Make an explicit dependency to the libs install targets would fix this
issue.

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 gdk-pixbuf/Makefile.am |    1 +
 libdeps.mk             |    3 +++
 2 files changed, 4 insertions(+), 0 deletions(-)
 create mode 100644 libdeps.mk

diff --git a/gdk-pixbuf/Makefile.am b/gdk-pixbuf/Makefile.am
index 95a93a8..db44cae 100644
--- a/gdk-pixbuf/Makefile.am
+++ b/gdk-pixbuf/Makefile.am
@@ -783,3 +783,4 @@ loaders.cache:
 endif
 
 -include $(top_srcdir)/git.mk
+-include $(top_srcdir)/libdeps.mk
diff --git a/libdeps.mk b/libdeps.mk
new file mode 100644
index 0000000..d7a10a8
--- /dev/null
+++ b/libdeps.mk
@@ -0,0 +1,3 @@
+# Extending dependencies of install-loaderLTLIBRARIES:
+# The $(lib-LTLIBRARIES) is needed by relinking $(loader_LTLIBRARIES)
+install-loaderLTLIBRARIES: install-libLTLIBRARIES
-- 
1.7.6.1

